code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -2,100 +2,134 @@
 
 /obj/item/weapon/storage/box/syndicate/New()
```

Comment:
I suggest moving the repeated "return" after the switch statement, instead of in each "case"